napalm_eos/eos.py

```diff
@@ -985,16 +985,20 @@ def get_mac_address_table(self):
     def get_route_to(self, destination='', protocol=''):
         routes = dict()
 
+        if protocol.lower() not in ['', 'bgp', 'connected', 'isis', 'ospf', 'rip', 'static']:
```

Comment:
My approach was to enumerate ALL protocols supported by the driver, and raise TypeError for a bad one, instead of having the underlying driver pyeapi raise a CommandError.  Either way works for me